R/dsl-storage.R

```diff
@@ -118,10 +118,15 @@ dsl_load <- function(path = "tic.R", force = FALSE, quiet = FALSE) {
 #' @rdname dsl_get
 #' @export
 dsl_init <- function(quiet = FALSE) {
+  path <- file.exists(paste0(usethis::proj_get(), "/tic.R"))
```

Comment:
`dsl_init()` is designed to always work, and is used in the examples. Can we leave it as is?